tests/resources/oracle_test_tables.sql

```diff
@@ -258,6 +258,32 @@ INSERT INTO pso_data_validator.dvt_char_id VALUES ('DVT4', 'Row 4');
 INSERT INTO pso_data_validator.dvt_char_id VALUES ('DVT5', 'Row 5');
 COMMIT;
 
+DROP TABLE pso_data_validator.dvt_fixed_char_id;
```

Comment:
Same as for BigQuery, the new tables appear to be duplicates of existing tables.